test/functional/apps/management/data_views/_scripted_fields.ts

```diff
@@ -44,7 +44,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     'unifiedFieldList',
   ]);
 
-  describe('scripted fields', function () {
+  describe.skip('scripted fields', function () {
```

Comment:
Would these tests pass if we define `data_views:cache_max_age` as `0` here?

Reply:
I'll try that. I'm sorting through this with the flaky test runner and this branch - https://github.com/elastic/kibana/pull/171265